### PR TITLE
{bio}[foss/2023b] purge_dups v1.2.6

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 24.04.1 LTS (Noble Numbat), x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.12.3
See https://gist.github.com/bedroge/05eabb465cb30417e26ce5654099c16e for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.10, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.6.8
See https://gist.github.com/bedroge/af62a66b7006dfe296167332e8508802 for a full test report.

File: easybuild/easyconfigs/p/purge_dups/purge_dups-1.2.6-foss-2023b.eb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        'start_dir': 'purge_dups-%(version)s',
+        'prebuildopts': "cd src && ",
+        'buildopts': 'CC="$CC" CFLAGS="$CFLAGS"',
+        'files_to_copy': ['bin', 'scripts'],
+    }),
+    ('KMC', '20190629', {
+        'source_urls': ['https://github.com/dfguan/KMC/archive'],
+        'sources': [{'download_filename': 'c05a5f9.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'patches': [{
+            'name': 'KMC_use_EB_pybind11_and_libs.patch',
```
Comment (bedroge) [inline easybuild/easyconfigs/p/purge_dups/purge_dups-1.2.6-foss-2023b.eb:41]: Shouldn't  this include the KMC version (and may some comment or other name like `KMC-dfguan` to indicate that this is for a fork of KMC, as it confused me for a while when I saw your other KMC PR :smile:)?

Comment (akesandgren) [reply easybuild/easyconfigs/p/purge_dups/purge_dups-1.2.6-foss-2023b.eb]: @bedroge Better now?

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 24.04.1 LTS (Noble Numbat), x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.12.3
See https://gist.github.com/bedroge/a6e68e69851b00e442892d37f86f5582 for a full test report.

**Review by bedroge — APPROVED** [review]
Lgtm

**Comment by bedroge** [comment]
Going in, thanks @akesandgren!

**Comment by Flamefire** [comment]
This fails for me because it tries to run the test step (`make`) in `purge_dups-1.2.6` instead of the `src` subdir

Any idea why this didn't fail before?

Edit: Tests steps for bundle are only run [recently](https://github.com/easybuilders/easybuild-easyblocks/pull/3748)

**Comment by ocaisa** [comment]
@Flamefire I think I have a general fix for this in https://github.com/easybuilders/easybuild-easyblocks/pull/3775

**Pull request merged**